### DISCUSSION TITLE
Create mods to increase OD and CS in a similar way to HR but separately

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#29131] [Ideas]
I want to propose the creation of 2 new mods that increases OD and CS separately by a fixed value in the same way that HR does. Both would apply a lower multiplier than Hard Rock (maybe 1.02x).
The OD increase mod would multiply the map OD by 1.4 and max at 10 like Hard Rock. And the same would happen with the CS increase mod, but multiplying the map CS by 1.3, again, like Hard Rock.

### Why create these mods?

I feel like it should be a good way to appease those that want to increase these difficulty settings separately using Difficulty Adjust, but without having to deal with the multiplier balancing complexity of Difficulty Adjust, since these mods would increase them using fixed value multipliers (at the moment, the same that are used by Hard Rock).

Edit: As stated on the comments, I think these mods could become useless if DA get a proper balance in the future. So I think this proposal should wait for a proper discussion about the DA multiplier and if it will ever get a good multiplier, because I guess I made this proposal expecting that DA could never receive a proper balance.

### Why not create mods for increasing AR and HP too?

AR is very hard to balance since, most of the time, it depends on what AR range the player is comfortable with and sometimes low AR might be harder than high AR due to reading difficulty.
As for HP, I would argue that it's not a much popular use case and it would probably receive a 1.0x multiplier since I don't see how it increases difficulty in any different way from Accuracy Challenge, Sudden Death or Perfect mod. But, since it would receive a 1.0x multiplier anyways, I don't see any problem with creating a mod for it if there is demand.

### Why use the same value multipliers as Hard Rock?

I wanted to use a safe value for the time being, but it might be higher as long as it doesn't increase mod multiplier balancing complexity by much. The goal is to create mods that give players that want to play with higher CS or OD a fair leaderboard position.

**Comment by snalgae** [comment]
Creating mods for individual difficulty values that increase them with a single multiplier is unnecessary when DA exists.
I don't think the lazer development team should forego the balancing of DA and instead implement multiplier increases of OD and CS.
What would happen to these mods when DA gets its score multiplier balancing?

**Reply by felipemarins** [reply]
The problem with DA getting a good multiplier is that there are many people that would oppose that for various reasons (like it being against mapper's intention and it allowing the player to fine tune the map). I don't necessarily agree with those reasons, but I feel like I made this discussion thinking more about the possibility of DA never getting a proper multiplier balance and I guess I should have written it on the discussion post. Maybe I should wait for when the devs ask for a proper discussion about DA multiplier to happen, because I agree that these mods could become useless if a proper balance of DA is well received.

**Comment by minisbett** [comment]
I think adding an extra mod that technically does the same thing just with different numbers is sort of pointless and bloats the mod menu too much.


However what I think would be pretty cool is a hardrock multiplier as a setting for the HR mod, that ranges from let's say 0.1 to 2, with 1 being your normal HR. And for the notes flipping, there could just be an extra option. PP wise there'd also be no challenge to implement it as everything only affects pre-processing.

**Reply by felipemarins** [reply]
I don't know how I feel about a "hardrock multiplier", maybe it could lead to scenarios where the difficulty settings of the map are already maximized but it still gives more score multiplier for increasing it. And it doesn't exactly solve the problem of players that want to increase OD or CS without affecting AR, unless you also add an option to disable AR increasing in HR, that way I could agree that maybe it's a better solution to avoid bloating mod select.

**Comment by Tenexxt** [comment]
In my opinion it would just be better to fix the difficulty adjust mod so even when you make the gameplay harder, you get less score + unranked.

**Reply by felipemarins** [reply]
Getting less score would not achieve the goal I mentioned on the discussion. And, as already said previously in the discussion, there might not be a simple fix for DA mod and there isn't a unanimous support by the community for fixing it. I made this discussion expecting that it might never be balanced properly in the future because of negative feedback and just be considered a not competitive mod.

**Reply by Tenexxt** [reply]
I musthave worded it incorectly, i meant that you would get a multiplier depending on the changes, not a fixed 0.50 multiplier.
I don't think OD and CS need a seperate mod, but ill leave the decision to the devs
